packages/block-library/src/columns/style.scss

```diff
@@ -10,41 +10,37 @@
 }
 
 .wp-block-column {
-	flex: 1;
+	flex-grow: 1;
 	margin-bottom: 1em;
 
 	// Responsiveness: Show at most one columns on mobile.
 	flex-basis: 100%;
```

Comment:
If we change this to `width:100%;` we accomplish the same thing and make it easier to override.

(content < width < flex-basis)